Source/Models/BrickDimension.swift

```diff
@@ -24,6 +24,16 @@ extension UIView {
     
 }
 
+public struct BrickSize {
+    let width: BrickDimension
```

Comment:
Can the width and the height variables of BrickSize be `var` instead of `let`. So if you want to change it, we don’t need to reconstruct the BrickSize?